ui/src/components/chat/compose/mod.rs

```diff
@@ -547,6 +570,37 @@ fn get_topbar_children(cx: Scope<ComposeProps>) -> Element {
         get_local_text("uplink.members"),
         all_participants.len()
     );
+
+    let conv_id = data.active_chat.id.clone();
```

Comment:
Just to avoid clippy, we can remove clone() here. 


```suggestion
    let conv_id = data.active_chat.id;
```